src/ansi-c/library/cprover_contracts.c

```diff
@@ -84,7 +99,7 @@ typedef struct
   __CPROVER_contracts_obj_set_t deallocated;
   /// \brief Pointer to object set supporting the is_fresh predicate checks
   /// (indexed mode)
-  __CPROVER_contracts_obj_set_ptr_t linked_is_fresh;
+  __CPROVER_contracts_ptr_pred_ctx_ptr_t linked_ptr_pred_ctx;
```

Comment:
The name is `ptr_pret_ctx` means  "pointer predicate context", it is not specific to is_fresh anymore (will be used by all predicates).